### PR TITLE
Update pagerank online documentation

### PR DESCRIPTION
**Pull request by Josipmrden**
### Description

Additional info added to online pagerank documentation to improve readability

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label

### CODE REVIEW

File: pages/advanced-algorithms/available-algorithms/pagerank_online.mdx [hunk 1]
```diff
@@ -48,10 +48,18 @@ its expectation, which is Rank(v).
 | **Edge weights**    | unweighted |
 | **Parallelism**     | sequential |
 
+<Callout type="info">
+This algorithm is currently running in a sequential manner, but can be parallelized.
+If you have a demand for parallelizing this algorithm, please reach us over
```
Comment (hal-eisen-MG) [inline pages/advanced-algorithms/available-algorithms/pagerank_online.mdx:53]: s/demand/use case
s/reach/contact

File: pages/advanced-algorithms/available-algorithms/pagerank_online.mdx [hunk 2]
```diff
@@ -48,10 +48,18 @@ its expectation, which is Rank(v).
...
+**[Discord](https://discord.gg/memgraph).**
+</Callout>
+
 ## Procedures
 
 Online PageRank should be used by executing the procedures in the following way:
-1. Parameters and the context is set using the `set()` procedure.
+1. With the `set()` procedure, the initial PageRank values are initialized on the nodes. This function
```
Comment (hal-eisen-MG) [inline pages/advanced-algorithms/available-algorithms/pagerank_online.mdx:60]: What value is used for initialization? Random? Zero?

File: pages/advanced-algorithms/available-algorithms/pagerank_online.mdx [hunk 3]
```diff
@@ -63,7 +71,9 @@ Online PageRank should be used by executing the procedures in the following way:
   SET node.rank = rank;
   ```
 
-3. Use the `get()` procedure to return the resulting values stored in the cache.
+3. Use the `get()` procedure to return the resulting values stored in the cache. If the user hasn't previously run
+`set()`, the procedure will also do the `set()` functionality prior in order to initialize the streaming context of
```
Comment (hal-eisen-MG) [inline pages/advanced-algorithms/available-algorithms/pagerank_online.mdx:75]: s/prior/first

File: pages/advanced-algorithms/available-algorithms/pagerank_online.mdx [hunk 4]
```diff
@@ -72,12 +82,15 @@ You can execute this algorithm on [graph projections, subgraphs or portions of t
 
 ### `set()`
 
-The procedure calculates PageRank for the nodes in the graph. 
+The procedure calculates PageRank for the nodes in the graph. The procedure is currently running in a 
+sequential manner, but can be parallelized 
+(the [non-streaming version of pagerank](/advanced-algorithms/available-algorithms/pagerank) offers 
+parallelism, but is not relevant as it doesn't set the streaming context for the algorithm).
```
Comment (hal-eisen-MG) [inline pages/advanced-algorithms/available-algorithms/pagerank_online.mdx:88]: s/relevant/applicable

File: pages/advanced-algorithms/available-algorithms/pagerank_online.mdx [hunk 5]
```diff
@@ -72,12 +82,15 @@ You can execute this algorithm on [graph projections, subgraphs or portions of t
...
 
 {<h4> Input: </h4>}
 
 - `subgraph: Graph` (**OPTIONAL**) ➡ A specific subgraph, which is an [object of type Graph](/advanced-algorithms/run-algorithms#run-procedures-on-subgraph) returned by the `project()` function, on which the algorithm is run.
-- `walks_per_node: integer (default=10)` ➡ Number of sampled walks per node.
+- `walks_per_node: integer (default=10)` ➡ Number of sampled walks per node. Note that increasing the number of walks per node will make the execution longer.
```
Comment (hal-eisen-MG) [inline pages/advanced-algorithms/available-algorithms/pagerank_online.mdx:93]: s/longer/run for more time

File: pages/advanced-algorithms/available-algorithms/pagerank_online.mdx (hunk 2 above)
Comment (Josipmrden) [reply pages/advanced-algorithms/available-algorithms/pagerank_online.mdx:60]: sorry, I formulated it differently

**Review by katarinasupe — APPROVED** [review]

**Pull request merged**